static/downloadables/runPrsCLI.sh

```diff
@@ -731,13 +781,36 @@ calculatePRS () {
                     echo "Check the value and try again."
                     exit 1
                 fi;;
+            b)  userGwasBeta=1 ;;
+            y)  valueType=$(echo "$OPTARG" | tr '[:upper:]' '[:lower:]')
+                if [ $valueType != 'beta' ] && [ $valueType != 'odds' ]; then
+                    echo "Invalid argument for -y. Use 'beta' and/or 'odds"
+                    echo -e "${LIGHTRED}Quitting...${NC}"
+                    exit 1
+                else
+                    #todo maybe say somewhere that if exclude is there, we will exclude calculations that use sex specific snps and that will trump any other inputs for the param
```

Comment:
Test this code:
if [ ${#sexForCalc[@]} -eq 0 ]; then
echo "'e' or 'exclude' present. Other sex parameters will be ignored."
fi